### DISCUSSION TITLE
Enabling 64-Bit support / Whisky macOS 14

### DISCUSSION THREAD

**Discussion by janerik20** [Whisky-App/Whisky#619] [Q&A]
Hi. I installed Whisky on macOS 14 Sonoma to run Windows Steam games on my MacBook Air M2 with 8GB RAM and 128GB Disk Space. I recognised that Steam installed the 32 Bit version in Whisky, how can I install the 64 Bit version? I think it will speed up my games a bit, because some like Portal 2 are a bit laggy.

<img width="718" alt="Screenshot 2023-11-14 at 21 41 52" src="https://github.com/Whisky-App/Whisky/assets/112584178/1a0b58ff-0ac4-4ca3-a493-14fc8d72fb51">

Thank you,
Jan

**Comment by Aem1024** [comment]
Afaik there is no 64-bit version of steam, Portal 2 is a 32 bit game.

**Reply by janerik20** [reply]
Well I know there is no 32 bit version of Portal 2... I thought I saw that in Windows there was a 64 Bit version installed... Can I install 64 Bit applications on Whisky out of the box?

And how could I speed up gaming a little bit, on other Macs it is faster.

**Comment by sheutran** [comment]
Hello janerik20,
I don't know if you are still looking for hints, but I recently discovered whisky + steam on my mac M2 (16Go unified memory) and I managed to play Portal 2 very well. The only bug I faced was when loading the game in 1920x1080 resolution, and since I dropped theses parameters down to 1280x720 (or a little in between) I had no more issues.  To do so, I just went into the option menu of Portal 2 then Video option. 
I did not tweak Whisky/Wine configurator from the day I installed a steam.exe bottle. When I had more serious trouble running another game, I looked at the 32 Vs 64bits steam application question. It seems that for any Windows operating system, steam.exe  is always 32bits. It only exists in 64bits for Mac Os.
